client/src/components/HandMarkedPaperBallot.tsx

```diff
@@ -261,23 +261,17 @@ const HandMarkedPaperBallot = ({
   useLayoutEffect(() => {
     const printBallot = printBallotRef?.current
 
-    if (!printBallot) {
-      return
-    }
-
-    ; (async () => {
-      const flow = await new Previewer().preview(
-        ballotRef.current!.innerHTML,
-        ['/ballot/ballot.css'],
-        printBallot
-      )
-      console.log('preview rendered, total pages', flow.total, { flow })
-      onRendered?.({ ballotStyleId, election, isLiveMode, precinctId, votes })
-    })()
+    new Previewer().preview(
+      ballotRef.current!.innerHTML,
+      ['/ballot/ballot.css'],
+      printBallot
```

Comment:
so you're okay if this is null? Or maybe it never was?

Reply:
I've added some logging to try and understand what happens… I'm not sure how it's working, but it is. 🤷‍♂️ I'm worried that I don't understand it.